### ISSUE TITLE
Problem with clickOutsideAction in webcomponent setup

### ISSUE THREAD

**Issue by nosovk** [gyurielf/svelte-tel-input#170]
I'm trying to create webcomponent that will wrap sveltecomponent to use it in html templates.
Here is a repro: https://github.com/nosovk/KBC-webcomponents here is testing page: https://kbc-webcomponents.vercel.app/lib/

Integration seems to be easy, except one problem.
<img width="758" alt="image" src="https://github.com/gyurielf/svelte-tel-input/assets/1398808/1c360857-28d6-4cbd-afb8-6ebf1b8eb112">
It seems that `clickOutsideAction` blocks another handler from firing ([handleSelect(country.iso2, e);](https://github.com/nosovk/KBC-webcomponents/blob/main/packages/lib/src/KBCDatepicker.wc.svelte#L184)). If I remove `clickOutsideAction` click handler works as expected.

Could you please modify `clickOutsideAction` to not prevent other events in a flow?

**Comment by nosovk** [comment]
probably move to https://github.com/sveltejs/site-kit/blob/master/packages/site-kit/src/lib/actions/click-outside.js can help?

**Comment by gyurielf** [comment]
I'll adjust it.  I make the prevent checking optional.

Otherwise, it's just a utility fn, not necessary to use or you can use your ow implementation.
The main purpose was provide all of the utilities which needs to make a proper phone input.

**Comment by nosovk** [comment]
hm, I found that I can create my own function, but it seems that any capture: true brokes clicks on language items

**Comment by gyurielf** [comment]
I walked around the issue, if I uncheck Ancestors, just the button event listeners remains.
Did you managed to fix this or need some help ?

**Comment by nosovk** [comment]
Nope, wasn't able to fix. I think that problem could be in custom element's wrapper, but wasn't able to confirm the actual reason of bug.
